<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>파일 찾기</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/rename.png</normaloff>../img/rename.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="chkAll">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>선택/해제</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtSearchKey"/>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="btnDoWork">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Do Work</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="btnStop">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QTableWidget" name="tblFiles">
       <column>
        <property name="text">
         <string>선택</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>파일구분</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>파일경로</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>파일명</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>수정일</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>생성일</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>최종사용일</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QPushButton" name="btnOk">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>파일명 변경</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>현재 경로 :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtUrl"/>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="cboWork">
       <item>
        <property name="text">
         <string>01.파일명 찾기</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>02.문자열로 파일 찾기</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblMsg">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
